erc20_moc/Bet.noblock.js

```diff
@@ -234,21 +236,20 @@ class Bet {
       }
       // TODO: implement a method to transfer several tokens in one call (like batch transfer)
       // get the collateral from provides
-      await this.collateralToken.transferChain(dbClient, provider, this.walletId, amount);
+      await this.collateralToken.transferChain(dbClient, provider, this.walletId, 'eth', 'bet', amount);
 
       // always mint the `amount` of every outcome tokens
       const tokens = this.getOutcomeTokens();
       for (const token of tokens) {
-        await token.mintChain(dbClient, this.walletId, amount);
+        await token.mintChain(dbClient, this.walletId, 'bet', amount);
       }
 
       // send back the tokens to make the market (new market) / keep the price constant (further adds)
       if (sendBacksAmounts) {
         for (const [o, a] of Object.entries(sendBacksAmounts)) {
           if (a > 0n) {
             const token = new ERC20(this.getOutcomeKey(o));
-            // console.log(`TX back ${o}: ${a}`);
-            await token.transferChain(dbClient, this.walletId, provider, a);
+            await token.transferChain(dbClient, this.walletId, provider, 'bet', 'eth', a, 'WFAIR');
```

Comment:
here you transfer outcome tokens. outcome tokens exist only within `bet` namespace. what `eth` is doing here?